apps/dashboard/src/pages/settings.tsx

```diff
@@ -0,0 +1,124 @@
+import { Card } from '@/components/primitives/card';
+import { Tabs, TabsContent, TabsList, TabsTrigger } from '@/components/primitives/tabs';
+import { OrganizationProfile, UserProfile } from '@clerk/clerk-react';
+import { DashboardLayout } from '../components/dashboard-layout';
+import { useNavigate, useLocation } from 'react-router-dom';
+import { ROUTES } from '@/utils/routes';
+
+export const clerkComponentAppearance = {
+  elements: {
```

Comment:
Please update the primary font color (black) to match the Dashboard primary font color `rgba(82, 88, 102, 0.95)`